member_type.go

```diff
@@ -0,0 +1,14 @@
+package zoom
+
+// Member represents an group member
+type Member struct {
+	ID    string `json:"id"`
+	Email string `json:"email"`
+}
+
+// ResopnseAddGroupMembers represents response for added member to group
+type ResopnseAddGroupMembers struct {
+	// IDs has comma-delimited, like 'xxxxxxxxxx,xxxxxxxxxx'
+	IDs     string `json:"ids"`
+	AddedAt string `json:"added_at"`
+}
```

Comment:
Are either of these types used in other API calls? If so, they can remain in this separate file, but if not, let's move them into group_member_post.go

Reply:
No used, I'll move to `group_member_post.go`